modules/client-configuration/pages/clients-and-activation-keys.adoc

```diff
@@ -30,31 +17,76 @@ An activation key can specify:
 
+image::provision-config-keys.png[scaledwidth=80%]
 
+.Procedure: Creating Activation Keys
+. Log in to the {productname} {webui} as an administrator, and navigate to menu:Systems[Activation Keys].
+. Click the btn:[Create Key] button.
+. On the [guimenu]``Activation Key Details`` page, in the [guimenu]``Description`` field, enter a name for the activation key.
+. In the [guimenu]``Key`` field, enter the distribution and service pack associated with the key.
+For example, ``SLES12-SP4`` for {sls}{nbsp}12{nbsp}SP4.
++
+[IMPORTANT]
+====
+Do not use commas in the [guimenu]``Key`` field for any {suse} products.
+However, you *must* use commas for Red Hat Products.
+For more information, see pass:c[xref:FILENAME.adoc#ref.webui.systems.activ-keys[]].
+====
++
+. In the [guimenu]``Base Channels`` drop-down box, select the appropriate base software channel, and allow the relevant child channels to populate.
+. Select the child channels you need (for example, the mandatory {susemgr} tools and updates channels).
++
+
+image::systems_create_activation_key_childchannels.png[scaledwidth=80%]
+
+. We recommend you leave the [guimenu]``Contact Method`` set to [guimenu]``Default``.
+. We recommend you leave the [guimenu]``Universal Default`` setting unchecked.
+. Click btn:[Update Activation Key] to create the activation key.
+. Check the [guimenu]``Configuration File Deployment`` check box to enable configuration management for this key, and click btn:[Update Activation Key] to save this change.
++
+[NOTE]
+====
+Note that the [guimenu]``Configuration File Deployment`` check box does not appear until after you have created the activation key.
+Ensure you go back and check the box if you need to enable configuration management.
+====
 
-=== Provisioning and Configuration
 
-.Provisioning and Configuration Overview
 
-image::provision-config-keys.png[scaledwidth=80%]
+== Activation Key Best Practices
+
+.Default Parent Channel
+
+Avoid using the [systemitem]``SUSE Manager Default`` parent channel.
+This setting forces {productname} to choose a parent channel that best corresponds to the installed operating system, which can sometimes lead to unexpected behavior.
+Instead, we recommend you create activation keys specific to each distribution and architecture.
+
+.Bootstrapping with Activation Keys
 
+If you are using bootstrap scripts, consider creating an activation key for each script.
+This will help you align channel assignments, package installation, system group memberships, and configuration channel assignments.
+You will also need less manual interaction with your system after registration.
 
+.Bandwidth Requirements
 
-=== Activation Keys Best Practices
+Using activation keys might result in automatic downloading of software at registration time, which might not be desirable in environments where bandwidth is constrained.
 
+These events create bandwidth usage:
```

Comment:
I would not call those "events", as they are really Activation Key configuration options.